storage/cloud-client/hmac.py

```diff
@@ -0,0 +1,154 @@
+# Copyright 2019 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Samples to illustrate management of HMAC keys via the python client library.
+"""
+
+
+from google.cloud import storage
+
+
+def list_keys(project_id):
+    """
+    List all HMAC keys associated with the project.
+    """
+    # [START storage_list_hmac_keys]
+    # project_id = 'Your Google Cloud project ID'
+    storage_client = storage.Client(project=project_id)
+    hmac_keys = storage_client.list_hmac_keys(project_id=project_id)
+    print('HMAC Keys:')
+    for hmac_key in hmac_keys:
+        print('The HMAC key metadata is:')
+        print('Key ID: {}'.format(hmac_key.id))
+        print('Access ID: {}'.format(hmac_key.access_id))
+        print('Project ID: {}'.format(hmac_key.project))
+        print('State: {}'.format(hmac_key.state))
+        print('Created At: {}'.format(hmac_key.time_created))
+        print('Updated At: {}'.format(hmac_key.updated))
+        print('Etag: {}'.format(hmac_key.etag))
+    # [END storage_list_hmac_keys]
+    return hmac_keys
+
+
+def create_key(project_id, service_account_email):
+    """
+    Create a new HMAC key using the given project and service account.
+    """
+    # [START storage_create_hmac_key]
+    # project_id = 'Your Google Cloud project ID'
+    # service_account_email = 'Service account used to generate HMAC key'
+    storage_client = storage.Client(project=project_id)
+    hmac_key, secret = storage_client.create_hmac_key(
+                                service_account_email=service_account_email,
+                                project_id=project_id)
+    print('The base64 encoded secret is {}'.format(secret))
+    print('Do not miss that secret, there is no API to recover it.')
```

Comment:
```suggestion
    print('Do not lose this secret, as it cannot be recovered.')
```

Reply:
This language was taken from the canonical sample, see https://github.com/googleapis/nodejs-storage/blob/master/samples/hmacKeyCreate.js#L48 . I would think that consistency is most important, unless we want to change it in all of them?

Reply:
I'll leave the final decision up to @frankyn, but my concerns are that:

1. It's currently improper english, which might make it harder for non-native english speakers to understand
2. Saying "there is no API to recover it" seems to imply that there may be some other way to recover it, like a support ticket or something (which I assume it not the case)